src/main/docker/bootstrap.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+cd /opt/Artemis
+
+if [ -z "$(ls -A config)" ]; then
+   echo "Config is Empty .. copying default ones .."
+   cp -n -a /defaults/Artemis/. config/
+else
+   echo "Config is not empty .. not copying default configs .."
+fi
```

Comment:
I don't think it is a good idea to introduce this complexity here. We should refactor the docker configuration so all values that have to be configured are handed to the container via the environment. We can still copy the `application.yml` to the container for all default values.

Reply:
What do you mean with "via environment" most of the configs are not configurable via environment.

Reply:
This does not copy the data to the container but to the config volume (or mapping) if the configs do not exist

Reply:
I think for easy use this might help a lot. Nevertheless, this part is something wie can easily drop.
If we drop this, we can simply delete https://github.com/ls1intum/Artemis/pull/4886#discussion_r839897758 as well :)